### ISSUE TITLE
about naming, consistency, name clashes, module organisation

### ISSUE THREAD

**Issue by rvion** [OutWatch/purescript-outwatch#7]
I love OutWatch's overall design, but have some issues with consistency, modules organisation, and naming clashes (I saw the latests re-organisation commits, but I think there is more to discuss)

Purescript
 - doens't support [qualified re-export](https://github.com/purescript/purescript/issues/2478) (and may never will)
 - has quite a strict import policy

names like `div` are really annoying to use because they clash with prelude.
name like `tpe` are also annoying to use because they are harder to remember

would it make sense to follow [lucid](http://chrisdone.com/posts/lucid#blaze) design and prefix or postfix everything with an `_` ?

 - no more Names that conflict with base (div, id, head, map) ->  only: div_; ul_, id_, etc.
 - no more names that are keywords, (type -> tpe or type_ or ..., class -> cls or class or ...) -> only class_, type_, etc.
 
it could also help adding back an global `OutWatch` that reexport most useful functions

(to see some more lucid examples, github link is here: https://github.com/chrisdone/lucid)

**Comment by LukaJCB** [comment]
Hey @rvion thanks a lot for bringing this up! :) 

I can definitely see the issues with the naming clashes in prelude and I'm not happy with it either.
However I'm not a hundred % convinced yet, that this is the answer. I can't really point to exactly why, but to me it's not pleasing solution yet.

Maybe, though, we could support both? Then you can import the module you like and go with that if you value consistency.

**Comment by rvion** [comment]
indeed, that's what I've been experimenting with, I'll share that very soon :)
